features/support/cucumber_helpers.rb

```diff
@@ -0,0 +1,59 @@
+# Makes it easy to build DRY Cucumber-compatible tables
```

Comment:
Please move this to `features/support/cucumber_table_builder.rb`.
I'd prefer to keep `features/supports/*_helpers.rb` as files that expose global functions. 
Any classes should live in their own file with the filename matching the class name.